### ISSUE TITLE
"ng add apollo-angular" for GraphQL doesn't seem to work anymore

### ISSUE THREAD

**Issue by ardianmaliqaj-git** [angular/angular-cli#22657]
### Which @angular/* package(s) are the source of the bug?

core

### Is this a regression?

Yes

### Description

"angular-apollo" seems to be incompatible with new Angular CLI changes.

### Please provide the exception or error you saw

```true
ng add apollo-angular

npm ERR! code ERESOLVE
npm ERR!
npm ERR! While resolving: angular@0.0.0
npm ERR! Found: @angular/core@13.1.3
npm ERR! node_modules/@angular/core
npm ERR!   @angular/core@"~13.1.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/core@"^6.1.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0" from apollo-angular@2.6.0
npm ERR! node_modules/apollo-angular
npm ERR!   apollo-angular@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by alan-agius4** [comment]
I'm sorry, but this issue is not caused by Angular CLI. Please contact the author(s) of the apollo-angular project or file an issue on their issue tracker.

**Comment by GUINCODE** [comment]
hello @ardianmaliqaj-git  ,
Are you found a solution for this issue  ?

**Comment by ardianmaliqaj-git** [comment]
Hi GUINCODE. Looks like apollo-angular team will have to work on making a compatible version of "apollo-angular" module for new version (v.13.) of Angular CLI.

The only thing I did was to uninstall angular cli, and to install the latest version of Angular 12 CLI, specificLly, that which I saw on Angular website.

Basically, this is not a bug, it's just that apollo-angular is not compatible with Angular v13
